app/femr/business/services/system/UpdatesService.java

```diff
@@ -0,0 +1,160 @@
+import femr.data.models.mysql.DatabaseStatus;
+
+import java.util.List;
+
+public class UpdatesService implements IUpdatesService {
+
+    private final IRepository<INetworkStatus> networkStatusRepository;
+    private final IRepository<IKitStatus> kitStatusRepository;
+    private final IRepository<IDatabaseStatus> databaseStatusRepository;
+
+    @Inject
+    public UpdatesService(IRepository<INetworkStatus> networkStatusRepository,
+                          IRepository<IKitStatus> kitStatusRepository,
+                          IRepository<IDatabaseStatus> databaseStatusRepository) {
+        this.networkStatusRepository = networkStatusRepository;
+        this.kitStatusRepository = kitStatusRepository;
+        this.databaseStatusRepository = databaseStatusRepository;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public ServiceResponse<List<? extends INetworkStatus>> retrieveNetworkStatuses() {
+        ServiceResponse<List<? extends INetworkStatus>> response = new ServiceResponse<>();
+        try {
+            List<? extends INetworkStatus> networkStatuses = networkStatusRepository.findAll(NetworkStatus.class);
+            response.setResponseObject(networkStatuses);
+
+        } catch (Exception ex) {
+            response.addError("", ex.getMessage());
+        }
+        return response;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public ServiceResponse<List<? extends INetworkStatus>> updateNetworkStatuses() {
+        ServiceResponse<List<? extends INetworkStatus>> response = new ServiceResponse<>();
+        //TODO: Lemur Team update the database with the right values
+        return response;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public ServiceResponse<List<? extends IKitStatus>> retrieveKitStatuses() {
+        ServiceResponse<List<? extends IKitStatus>> response = new ServiceResponse<>();
+        try {
+            List<? extends IKitStatus> kitStatuses = kitStatusRepository.findAll(KitStatus.class);
+            response.setResponseObject(kitStatuses);
+
+        } catch (Exception ex) {
+            response.addError("", ex.getMessage());
+        }
+        return response;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public ServiceResponse<List<? extends IKitStatus>> updateKitStatuses() {
+        ServiceResponse<List<? extends IKitStatus>> response = new ServiceResponse<>();
+        //TODO: Lemur Team update the database with the right values
+        return response;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public ServiceResponse<List<? extends IDatabaseStatus>> retrieveDatabaseStatuses() {
+        ServiceResponse<List<? extends IDatabaseStatus>> response = new ServiceResponse<>();
+        try {
+            List<? extends IDatabaseStatus> kitStatuses = databaseStatusRepository.findAll(DatabaseStatus.class);
+            response.setResponseObject(kitStatuses);
```

Comment:
Shouldn’t this variable be named databaseStatuses?

Reply:
yes sir, gotta change this